test/com/facebook/buck/apple/testdata/apple_test_xcuitest/FooXCUITest.m

```diff
@@ -0,0 +1,8 @@
+#import <XCTest/XCTest.h>
+
+@interface FooXCTest : XCTestCase
+@end
+
+@implementation FooXCTest
+
```

Comment:
Currently there are no tests here. And Buck prints out `NOTESTS`. Which is right-ish. I think what you want is to have some tests here, and then test that Buck actually skips them and still outputs `NOTESTS` for this file.